src/Gui/MainWindow.cpp

```diff
@@ -1917,13 +1917,15 @@ void MainWindow::startSplasher()
             GetGroup("BaseApp")->GetGroup("Preferences")->GetGroup("General");
         // first search for an external image file
         if (hGrp->GetBool("ShowSplasher", true)) {
-            d->splashscreen = new SplashScreen(this->splashImage());
+            d->splashscreen = new SplashScreen(this->splashImage(), 
+                Qt::WindowStaysOnTopHint | Qt::SplashScreen | Qt::X11BypassWindowManagerHint);
 
             if (!hGrp->GetBool("ShowSplasherMessages", false)) {
                 d->splashscreen->setShowMessages(false);
             }
 
             d->splashscreen->show();
+            QTimer::singleShot(3000, this, SLOT(stopSplasher()));
```

Comment:
Can you update to use the current functor-based connection instead? Thanks!

Reply:
Ok. changed.